assets/src/blocks/Submenu/SubmenuFrontend.js

```diff
@@ -1,8 +1,6 @@
 import { Component, Fragment } from '@wordpress/element';
-import { getSubmenuStyle, addSubmenuActions } from './submenuFunctions';
+import { getSubmenuStyle, addSubmenuActions, loadMenuItems } from './submenuFunctions';
 const { __ } = wp.i18n;
-const { apiFetch } = wp;
-const { addQueryArgs } = wp.url;
 
 export class SubmenuFrontend extends Component {
```

Comment:
This could easily be turned in a function component as the class doesn't have state. But we could do that in https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/328/files too

Reply:
Oh it does but the lines were hidden in the diff :facepalm: